docs/Developers/Review.rst

```diff
@@ -140,6 +140,9 @@ For the Reviewers
    Bears require special attention during testing.
    `Testing Bears <http://api.coala.io/en/latest/Developers/Testing_Bears.html>`__
    provides a guideline for how to test bears.
+-  Check that the changes in the PR haven't been pushed from the master
+   branch.If it is so, then drop a comment saying that it is not a good
+   practice, but also indicate that they don't need to close the PR for now.
```

Comment:
The following sentence would convey the meaning in a better way:
`close the PR for now` --> `close this particular PR and should take care of this from the next PR`